Add cross-namespace configuration info in op guide (#4691)
istio/istio.io: content/docs/ops/traffic-management/deploy-guidelines/index.md

This section provides specific deployment or configuration guidelines to avoid networking or traffic management issues.

## Cross-namespace configuration sharing

A `VirtualService`, `DestinationRule`, or `ServiceEntry` configuration resource
can be defined in one namespace and then reused in other namespaces, if it is exported.
All traffic management resources are exported to all namespaces by default,
but the visibility can be overridden using the `exportTo` field.
For example, the following virtual service can only be used by clients in the same namespace
as the configuration:

{{< text yaml >}}
apiVersion: networking.istio.io/v1alpha3
kind: VirtualService
metadata:
  name: myservice
spec:
  hosts:
  - myservice.com
  exportTo:
  - "."
  http:
  - route:
    - destination:
        host: myservice
{{< /text >}}

{{< tip >}}
Visibility of a Kubernetes `Service` can similarly be controlled using the `networking.istio.io/exportTo` annotation.
{{< /tip >}}

The visibility of a `DestinationRule` resource in a particular namespace does not

> guarantee it will be used. Exporting a destination rule to other namespaces enables it to be used

Reviewer: ```suggestion
guarantee the rule is enforced. Exporting a destination rule to other namespaces enables you to use it
```
